Travis "ERROR collecting psychopy/tests/test_all_visual/test_textbox.py"

My recent pull requests are failing because:

```
_______ ERROR collecting psychopy/tests/test_all_visual/test_textbox.py ________
psychopy/tests/test_all_visual/test_textbox.py:4: in <module>
    from psychopy.visual.textbox import TextBox
psychopy/visual/textbox/__init__.py:37: in <module>
    from .fontmanager import FontManager
psychopy/visual/textbox/fontmanager.py:21: in <module>
    from freetype import Face, FT_LOAD_RENDER, FT_LOAD_FORCE_AUTOHINT, FT_Exception
../../../miniconda/envs/psychopy-conda/lib/python2.7/site-packages/freetype/__init__.py:21: in <module>
    from freetype.raw import *
../../../miniconda/envs/psychopy-conda/lib/python2.7/site-packages/freetype/raw.py:249: in <module>
    FT_Outline_Done_Internal       = _lib.FT_Outline_Done_Internal
../../../miniconda/envs/psychopy-conda/lib/python2.7/ctypes/__init__.py:379: in __getattr__
    func = self.__getitem__(name)
../../../miniconda/envs/psychopy-conda/lib/python2.7/ctypes/__init__.py:384: in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
E   AttributeError: /home/travis/miniconda/envs/psychopy-conda/lib/python2.7/site-packages/wx-3.0-gtk2/wx/../../../.././libfreetype.so.6: undefined symbol: FT_Outline_Done_Internal****
```

--- a/conda/environment-2.7.yml
+++ b/conda/environment-2.7.yml
@@ -13,7 +13,7 @@ dependencies:
 - esprima-python
 - gitpython
 - ffmpeg
-- freetype-py
+- freetype-py<2.10.0
 - future
 - gevent
 - greenlet

--- a/conda/environment-3.6.yml
+++ b/conda/environment-3.6.yml
@@ -13,7 +13,7 @@ dependencies:
 - esprima-python
 - gitpython
 - ffmpeg
-- freetype-py
+- freetype-py<2.10.0
 - future
 - gevent
 - greenlet

--- a/conda/environment-3.7.yml
+++ b/conda/environment-3.7.yml
@@ -13,7 +13,7 @@ dependencies:
 - esprima-python
 - gitpython
 - ffmpeg
-- freetype-py
+- freetype-py<2.10.0
 - future
 - gevent
 - greenlet

--- a/setup.cfg
+++ b/setup.cfg
@@ -59,7 +59,7 @@ install_requires =
     gitpython
     astunparse
     esprima
-    freetype-py
+    freetype-py<2.10.0
     # Platform-specific dependencies.
     imageio < 2.5; python_version < "3"
     imageio >= 2.5; python_version >= "3"

